plugins/hid/hiddmxdevice.h

```diff
@@ -117,7 +129,8 @@ class HIDDMXDevice : public HIDDevice
     {
```

Comment:
Here I went for minimal change. We could also reflect the devices' internal bitmap here. That would be a change to:

```
    {
        DMX_MODE_NONE   = 1 << 0,
        DMX_MODE_MERGER = 1 << 1,
        DMX_MODE_OUTPUT  = 1 << 2,
        DMX_MODE_INPUT  = 1 << 3
    };
```

This way `m_mode` could be written to the device as is, saving us from fiddling around with ifs in [#plugins/hid/hiddmxdevice.cpp](https://github.com/mcallegari/qlcplus/blob/b8d7cb10dafb24159ea6ba2e21524b798ec48eb0/plugins/hid/hiddmxdevice.cpp#L227C1-L232C26)

I could rework this part to make it cleaner. Just now it's "less intrusive".